README.md

```diff
@@ -12,27 +18,37 @@ developed on new feature branches, and pull requests should be sent to
 the `dev` branch for code review. Merges into `master` happen at the end
 of sprints, and tags in `master` correspond to production releases.
 
-### Running Locally
 
-_Note: The following steps assume you have already installed Python (>=3.5) 
-on a system running Ubuntu 16.04 and higher, or OSX. OSX users will
-need [Docker for Mac](https://www.docker.com/docker-mac) installed, as well
-as Homebrew, for to `brew install iproute2mac git-crypt git-lfs`._
+### Installation
 
-Run `./install.sh` to set up the virtualenv. It will activate the `dr_env`
-for you the first time. This virtualenv is valid for the entire `data_refinery`
-repo. Sub-projects each have their own virtualenvs which are managed by their
-containers. When returning to this project you should run
-`source dr_env/bin/activate` to reactivate the virtualenv.
```

Comment:
I'm not sure I should actually take this out, but the virtualenv is pretty much only installing `pip-tools` so I'm not sure if its worth it or not.

Now that I think about it a bit more I think leaving it in is probably the right thing to do. Do you agree?

Reply:
Yes, I say keep this.